courses/machine_learning/deepdive2/introduction_to_tensorflow/solutions/3_keras_sequential_api.ipynb

```diff
@@ -22,9 +22,17 @@
   },
```

Comment:
<p>Any reason you setting steps_per_epoch manually to 1? It should be the value stored in the step_per_epoch variable, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/693/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>